products/contao

```diff
@@ -0,0 +1,38 @@
+---
+title: Contao
+category: server-app
+changelogTemplate: "https://github.com/contao/contao/blob/__RELEASE_CYCLE__/CHANGELOG.md"
+auto:
+  - git: https://github.com/contao/contao.git
+
+releases:
+  - releaseCycle: "5.0"
+    releaseDate: 2022-08-18
+    support: 2023-02-14
+    eol: 2023-02-14
+    latest: "5.0.7"
+    latestReleaseDate: 2022-11-30
+    lts: false
+    discontinued: false
+  - releaseCycle: "4.13"
+    releaseDate: 2022-02-17
+    support: 2025-02-14
+    eol: 2026-02-14
+    latest: "4.13.14"
+    latestReleaseDate: 2022-11-30
+    lts: true
+    discontinued: false
+
+iconSlug: NA
+permalink: /contao
+releasePolicyLink: https://contao.org/en/release-plan.html
```

Comment:
Seems like https://contao.org/release-plan.html is working and redirect to the user language (or at least the english language). https://contao.org/release-plan.html should be preferred over the 'localized' link.

Reply:
Done.